Porting to DSL2 (#42)
qbic-pipelines/bamtofastq: docs/README.md

> # nf-core/bamtofastq: Documentation

Reviewer: ```suggestion
# qbic-pipelines/bamtofastq: Documentation
```

> The nf-core/bamtofastq documentation is split into the following pages:

Reviewer: ```suggestion
The qbic-pipelines/bamtofastq documentation is split into the following pages:
```

- [Usage](usage.md)
  - An overview of how the pipeline works, how to run it and a description of all of the different command-line flags.
- [Output](output.md)
  - An overview of the different results produced by the pipeline and how to interpret them.

> You can find a lot more documentation about installing, configuring and running nf-core pipelines on the website: [https://nf-co.re](https://nf-co.re)

Reviewer: ```suggestion
You can find a lot more documentation about installing, configuring and running nf-core pipelines on the website: [https://nf-co.re](https://nf-co.re). This pipeline follows the nf-core template making the nf-core documentation compatible.
```
